cli/src/util/spm.ts

```diff
@@ -29,14 +40,65 @@ export async function findPackageSwiftFile(config: Config): Promise<string> {
 export async function generatePackageFile(config: Config, plugins: Plugin[]): Promise<void> {
   const packageSwiftFile = await findPackageSwiftFile(config);
   try {
+    logger.info('Writing Package.swift');
+
     const textToWrite = await generatePackageText(config, plugins);
     writeFileSync(packageSwiftFile, textToWrite);
   } catch (err) {
     logger.error(`Unable to write to ${packageSwiftFile}. Verify it is not already open. \n Error: ${err}`);
   }
 }
 
-async function generatePackageText(config: Config, plugins: Plugin[]): Promise<string> {
+export async function processIosPackages(config: Config): Promise<Plugin[]> {
+  const plugins = await getPlugins(config, 'ios');
+  const iosPlugins = await getIOSPlugins(plugins);
+  printPlugins(iosPlugins, 'ios', 'capacitor');
+
+  const packageSwiftPluginList = await pluginsWithPackageSwift(iosPlugins);
+  printPlugins(packageSwiftPluginList, 'ios', 'packagespm');
+
+  if (iosPlugins.length == packageSwiftPluginList.length) {
+    logger.info('Number of plugins in lists match'); // TODO: Word this better
```

Comment:
is this something you are going to address now?

Reply:
Yeah - let me fix that.